parser/break.c

```diff
@@ -0,0 +1,75 @@
+#include "break.h"
+#include <stdlib.h>
+#include <string.h>
+
+#include "parser.h"
+
+void break_debug(int depth, parser_node *node)
+{
+    printtabs(depth);
+    printf("Break\n");
+}
+
+apply_result *break_apply(parser_node *node, context *ctx)
+{
+    char* loop_end_label = (char*)malloc(128);
+    get_current_loop_end_label_counter(ctx, loop_end_label);
+    add_text(ctx, "jmp %s", loop_end_label);
+    free(loop_end_label);
+    return NULL;
+}
+
+parser_node *parse_break(typed_token **tkns_ptr)
+{
+    typed_token *tkn = *tkns_ptr;
+    if (tkn->type_id == TKN_ID)
```

Comment:
We need a separate token for `break` and `continue`! Add them to `lexer.h`/`lexer.c`